rotkehlchen/tests/unit/decoders/test_weth.py

```diff
@@ -465,3 +490,581 @@ def test_wxdai_wrap(database, gnosis_inquirer, gnosis_accounts):
     ]
 
     assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('arbitrum_one_accounts', [['0xBE6660FBE96B61B72Bf35FFaB40eB2CA886A7f85']])
+def test_weth_withdraw_arbitrum_one(database, arbitrum_one_inquirer):
+    evmhash = deserialize_evm_tx_hash('0xc19c7e1e0af7819b1922a287d034540e8f8dba4e065317d6483d48ac27e727e9')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=arbitrum_one_inquirer,
+        database=database,
+        tx_hash=evmhash,
+    )
+    timestamp = TimestampMS(1712238368000)
+    amount, gas_fees = '0.00052', '0.00000108547'
+    expected_events = [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_fees)),
+            location_label='0xBE6660FBE96B61B72Bf35FFaB40eB2CA886A7f85',
+            notes=f'Burned {gas_fees} ETH for gas',
+            counterparty=CPT_GAS,
+        ),
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.RETURN_WRAPPED,
+            asset=A_WETH_ARB,
+            balance=Balance(amount=FVal(amount)),
+            location_label='0xBE6660FBE96B61B72Bf35FFaB40eB2CA886A7f85',
+            notes=f'Unwrap {amount} WETH',
+            counterparty=CPT_WETH,
+            address=string_to_evm_address('0x82aF49447D8a07e3bd95BD0d56f35241523fBab1'),
+        ),
+        EvmEvent(
```

Comment:
Everywhere in all the tests you touches please use them like this. Saves lines.

```suggestion
        ), EvmEvent(
```